src/wagmi.ts

```diff
@@ -1,10 +1,10 @@
 import { http, cookieStorage, createConfig, createStorage } from "wagmi";
-import { mainnet, optimism, sepolia } from "wagmi/chains";
+import { mainnet, optimism, sepolia, optimismSepolia } from "wagmi/chains";
```

Comment:
we can probably forgo mainnet and sepolia altogether here, no need to support them